### PR TITLE
Bugs/misc

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: fuse/all.py [hunk 1]
```diff
@@ -290,7 +290,7 @@ def fuse_award():
     app_df.to_csv(deba.data("fuse/appeals.csv"), index=False)
     uof_citizen_df.to_csv(deba.data("fuse/uof_citizens.csv"), index=False)
     uof_officer_df.to_csv(deba.data("fuse/uof_officers.csv"), index=False)
-    award_df.to_csv(deba.data("fuse/award.csv"), index=False)
+    award_df.to_csv(deba.data("fuse/awards.csv"), index=False)
```
Comment (pckhoi) [inline fuse/all.py:293]: Perhaps you need to fix the filename in the `.wrgl/config.yaml` file as well?

**Review by pckhoi — COMMENTED** [review]
This PR seems to be mainly about changing `tracking_number` to `tracking_id`. I have zero problem with that but we need to coordinate with the web team. @rajivsinclair Is the web team aware of this change?

File: fuse/all.py (hunk 1 above)
Comment (ayyubibrahimi) [reply fuse/all.py:293]: Yes. Thank you!

**Comment by ayyubibrahimi** [comment]
@pckhoi the `canonicalize_officers` function that you worked on doesn't seem to be working as intended. I received the following:

```
    Traceback (most recent call last):
      File "/mnt/c/Users/ayyubi/lleads/processing/match/new_orleans_so.py", line 270, in <module>
        cprr19 = deduplicate_cprr_19_personnel(cprr19)
      File "/mnt/c/Users/ayyubi/lleads/processing/match/new_orleans_so.py", line 25, in deduplicate_cprr_19_personnel
        matcher = ThresholdMatcher(
      File "/home/ayyubibrahimi/.local/lib/python3.9/site-packages/datamatch/matchers.py", line 84, in __init__
        self._pairer = DeduplicatePairer(dfa, index)
      File "/home/ayyubibrahimi/.local/lib/python3.9/site-packages/datamatch/pairers.py", line 148, in __init__
        raise ValueError(
    ValueError: Dataframe index contains duplicates.
```

**Comment by pckhoi** [comment]
Yes, please. If there is a bug we can simply add more tests. Let me know if I should take a look as well.

**Comment by pckhoi** [comment]
@ayyubibrahimi let me know when this is ready for review.

**Comment by ayyubibrahimi** [comment]
Ready! Thanks.

**Pull request merged**